ricecooker/config.py

```diff
@@ -14,13 +14,78 @@
 SUSHI_BAR_CLIENT = None
 STAGE = False
 
-# Setup logger
+# Don't use this - call logging.getLogger(__name__) from each
```

Comment:
I'm aware this is the standard practices for python logging, but the convention for chef scripts has been to use `ricecooker.config.LOGGER` as the main logging interface (all logs combine in the same default logger, with log level set based on --warn, --debug, etc command flags).

Logging to `config.LOGGER` is a convenience method that saves chef-authors from the need to call setupLogging etc.

Reply:
The new `setup_logging` intends to do the same, it just has a default configuration that will work no matter the name of the log. Module paths are really convenient for understanding the origin of a log message.